e2e/tests/ui_tests/default_judgement/1v1UnspecDJ_test.js

```diff
@@ -39,8 +39,17 @@ Scenario('DefaultJudgement @create-claim @e2e-1v1-dj @e2e-wa @non-prod-e2e-ft',
     const caseProgressionTakeCaseOfflineTask = await api.retrieveTaskDetails(config.hearingCenterAdminWithRegionId1, caseId, config.waTaskIds.listingOfficerCaseProgressionTask);
     console.log('caseProgressionTakeCaseOfflineTask...' , caseProgressionTakeCaseOfflineTask);
   }
+
   await I.login(config.hearingCenterAdminWithRegionId1);
-  await I.staffPerformDJCaseTransferCaseOffline(caseId);
+  if (['preview', 'demo'].includes(config.runningEnv)) {
```

Comment:
else block should still execute await I.staffPerformDJCaseTransferCaseOffline(caseId);